CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.36.18 (2021-08-31)
+### Improvements
+- **Button**: updating small size button styling
```

Comment:
you should add your github name into brackets next to your change. Pastanaga is an open source repo and it's a common practice in open source projects ;)

Reply:
done